### DISCUSSION TITLE
Docker Deamon is not starting

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#404] [Q&A] [answered]
Hello,

I have problem with docker deamon auto-start on OVP800. I have restarted several times VPU. But still it shows Docker deamon is not running. Please see attached image.
![image](https://github.com/ifm/ifm3d/assets/110673255/0f003804-37e1-4601-a728-7fe2b627e01d)

**Comment by Astronomia-maker** [comment]
It is showing storage is full. But without root access i cannot delete some data. `/dev/mmcblk0p1  6.5G 6.4G 0 100% /` 
![image](https://github.com/ifm/ifm3d/assets/110673255/6a75b5b8-768a-44ce-ba89-4d66bf752f56)

**Reply by graugans** [reply]
Which firmware version are you running on the OVP800? By (re)installing a new firmware the the occupied space will be freed up again.

**Reply by Astronomia-maker** [reply]
I have firmware version 0.16.23-823. The reinstalling suggestion works I just did it. But is there any other command to freed up? While application at customer then reinstalling firmware and again installing our application could be little hectic.

**Reply by graugans** [reply, marked as answer]
Starting with firmware version 1.0.x the factory reset will also remove the Docker images. But it also frees any files on the `oem` folder. The benefit of the 1.0.x firmware is that it provides more free space so one can use bigger Docker files.

In general it may be of good advice to use the `journald` [logging driver](https://docs.docker.com/config/containers/logging/journald/) for the Docker containers otherwise the logs may fill up your VPU. If you stick with 0.16.x
